tests/testthat/_snaps/simulate-infections.md

```diff
@@ -2,67 +2,67 @@
-    24: reported_cases 2023-01-10 169.00000
-    25: reported_cases 2023-01-11 144.00000
-    26: reported_cases 2023-01-12 109.00000
-    27: reported_cases 2023-01-13  80.00000
-    28: reported_cases 2023-01-14  95.00000
+     1:     infections 2023-01-01 100.00000
+     2:     infections 2023-01-02 120.00000
+     3:     infections 2023-01-03 144.00000
+     4:     infections 2023-01-04 172.80000
+     5:     infections 2023-01-05 207.36000
+     6:     infections 2023-01-06 248.83200
+     7:     infections 2023-01-07 298.59840
+     8:     infections 2023-01-08 238.87872
+     9:     infections 2023-01-09 191.10298
+    10:     infections 2023-01-10 152.88238
+    11:     infections 2023-01-11 122.30591
+    12:     infections 2023-01-12  97.84472
+    13:     infections 2023-01-13  78.27578
+    14:     infections 2023-01-14  62.62062
+    15: reported_cases 2023-01-01 105.00000
+    16: reported_cases 2023-01-02 112.00000
+    17: reported_cases 2023-01-03 166.00000
+    18: reported_cases 2023-01-04 188.00000
+    19: reported_cases 2023-01-05 211.00000
+    20: reported_cases 2023-01-06 277.00000
+    21: reported_cases 2023-01-07 304.00000
+    22: reported_cases 2023-01-08 231.00000
+    23: reported_cases 2023-01-09 170.00000
+    24: reported_cases 2023-01-10 140.00000
+    25: reported_cases 2023-01-11 120.00000
+    26: reported_cases 2023-01-12  90.00000
+    27: reported_cases 2023-01-13  66.00000
+    28: reported_cases 2023-01-14  77.00000
               variable       date     value
 
 # simulate_infections works as expected with additional parameters
 
               variable       date     value
                 <char>     <Date>     <num>
-     1:     infections 2023-01-01 101.60082
-     2:     infections 2023-01-02 107.29011
-     3:     infections 2023-01-03 113.03527
-     4:     infections 2023-01-04 119.01464
-     5:     infections 2023-01-05 125.28049
-     6:     infections 2023-01-06 131.63823
-     7:     infections 2023-01-07 138.33602
-     8:     infections 2023-01-08  96.91916
-     9:     infections 2023-01-09  94.05215
-    10:     infections 2023-01-10  89.94060
-    11:     infections 2023-01-11  85.64594
-    12:     infections 2023-01-12  81.40206
-    13:     infections 2023-01-13  77.28891
-    14:     infections 2023-01-14  73.33458
-    15: reported_cases 2023-01-01  75.00000
-    16: reported_cases 2023-01-02  67.00000
-    17: reported_cases 2023-01-03  73.00000
-    18: reported_cases 2023-01-04  63.00000
-    19: reported_cases 2023-01-05 276.00000
-    20: reported_cases 2023-01-06 121.00000
-    21: reported_cases 2023-01-07 215.00000
-    22: reported_cases 2023-01-08  84.00000
-    23: reported_cases 2023-01-09 116.00000
-    24: reported_cases 2023-01-10 132.00000
-    25: reported_cases 2023-01-11  91.00000
-    26: reported_cases 2023-01-12  77.00000
-    27: reported_cases 2023-01-13 120.00000
-    28: reported_cases 2023-01-14  41.00000
+     1:     infections 2023-01-01  59.65590
```

Comment:
really quite a big change here?

Reply:
to be revisited once simulation code has been updated

Reply:
this is changed now but still quite different; it's because of the updated initial growth calculation I think